__tests__/page-snapshot-user-settings.tsx

```diff
@@ -0,0 +1,30 @@
+import User from 'pages/users/[id]'
+import { screen, waitForElementToBeRemoved } from '@testing-library/react'
+import { render } from 'jest.utils'
+import fetch from 'jest-fetch-mock'
+import { mockUserPage } from './page-snapshot-user'
+
+beforeEach(() => {
+  fetch.resetMocks()
```

Comment:
why do you need to reset mocks before every test? there's only one test

Reply:
Incorrect, the mocks are persisted across runs.